draft-packages/hero-card/KaizenDraft/HeroCard/HeroCard.scss

```diff
@@ -1,6 +1,6 @@
-@import "~@kaizen/design-tokens/sass/color-vars";
-@import "~@kaizen/design-tokens/sass/border-vars";
-@import "~@kaizen/design-tokens/sass/shadow-vars";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/shadow";
```

Comment:
What happened here? (It looks like this commit was about fixing rgba these lines revert imports to non-variable versions, and it's not clear why)

Reply:
Looks like it got fixed by the end of the PR 🤷